app/models/firm.rb

```diff
@@ -12,6 +12,11 @@ class Firm < ActiveRecord::Base
   ]
 
   scope :registered, -> { where.not(email_address: nil) }
+  scope :sorted_by_registered_name, -> { order(:registered_name) }
+
+  def self.most_recently_updated(limit: nil)
+    order(updated_at: :desc).limit(limit)
+  end
```

Comment:
Made the 2nd one a class method to keep Rubocop happy: it was too long as a proc and [this](https://github.com/bbatsov/ruby-style-guide/issues/270) ...